process.go

```diff
@@ -21,7 +17,4 @@ type ProcessConfig struct {
 	Stdin  io.Reader
 	Stdout io.Writer
 	Stderr io.Writer
-
-	// Console is the path to the pty slave for use by the master
-	Console string
```

Comment:
Could you explain why Console and Env should be only in a container config? I have only one explanation that nobody asked about this functionality.  If it's the reason, it's ok.